server/routes/v1/post.ts

```diff
@@ -27,6 +27,7 @@ postRouter.get('', async (req: Request, res: Response) => {
     return res.status(result.status).json(result);
   } catch (err) {
     console.error(err);
+    res.status(500);
```

Comment:
[Already sets a status message](https://github.com/GDSCUTM-CommunityProjects/UBoard/blob/main/server/middleware/auth.ts#L23). Failing to get the user shouldn't be a 500.

Here and elsewhere

Reply:
Which other ones? I see the getPost route also contains getauthuser, but we still need a fallback error code. Going to the route with an invalid postid will not return a status, so the frontend just gets stuck waiting for a response.